lib/uniffi-zcash/build.rs

```diff
@@ -0,0 +1,80 @@
+use std::fs::{read_to_string, OpenOptions};
+use std::io::Write;
+use std::path::Path;
+
+fn main() {
+    println!("cargo:rerun-if-changed=src/udl");
+    generate_udl_file();
+
+    uniffi::generate_scaffolding("./src/zcash.udl").unwrap();
+}
+
+fn generate_udl_file() {
+    let mut content =
+        "/* Autogenerated by the `build.rs` script - do not modify */\n\n".to_string();
+
+    content.push_str(&join_udl_files(&[
+        "src/udl/namespace.udl",
+        "src/udl/error.udl",
+        "src/udl/payment.udl",
+        "src/udl/hdwallet",
+        "src/udl/orchard",
+        "src/udl/zcash_client_backend",
+        "src/udl/zcash_primitives",
+    ]));
```

Comment:
Maybe we could use some globbing library that would select all files there automatically, so we would not need to edit this list manually on new files.

Reply:
Done.